packages/design/src/Elevations.mdx

```diff
@@ -11,7 +11,7 @@ The following table lists all elements that has a z-index specified on it.
 export function Value({ of: type }) {
   const styles = getComputedStyle(document.documentElement);
   const getValue = styles.getPropertyValue(`--elevation-${type}`);
-  return <>{getValue}</>;
+  return <span>{getValue}</span>;
```

Comment:
You likely tested this already but, are you able to use `React.Fragment` here?

I wonder if we could also just return `getValue`?

Reply:
I did not try the `<React.Fragment>` and it worked! Thanks for that